k8s/containers/seed-update/Makefile

```diff
@@ -0,0 +1,13 @@
+SHELL=/bin/bash
+buildversion = $(shell cat version)
+
+.PHONY: all
+all: build
+
+.PHONY: increment
+increment:
+	echo $$(( $(buildversion) + 1 )) > version
```

Comment:
This was just some boiler plate I use from other project build - I am not suggesting that Jenkiins uses the make file, it was just for local dev as I had to do manual builds because Jenkins is not pushing this to docker hub.